CODEOWNERS

```diff
@@ -0,0 +1,46 @@
+# These owners will be the default owners for everything in the repo.
+* @cramforce @dvoytenko
+
+# core javascript owners
+*.js @jridgewell @cramforce @dvoytenko
+
+# core md (docs) owners
+*.md @mrjoro @bpaduch
+
+# top level file owners
```

Comment:
Does it make sense to add `package.json` to this list, since version number changes (like the recent node change) could break code?